AmplifyPlugins/DataStore/AWSDataStoreCategoryPlugin/Sync/Events/SyncQueriesStartedEvent.swift

```diff
@@ -0,0 +1,18 @@
+//
+// Copyright 2018-2020 Amazon.com,
+// Inc. or its affiliates. All Rights Reserved.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+import Foundation
+
+/// Used as HubPayload for the `SyncQueriesStarted`
+public struct SyncQueriesStartedEvent {
+    /// list of model names
```

Comment:
"A list of all model names for which DataStore has started establishing subscriptions"

Reply:
Updated